### ISSUE TITLE
CRAB does not send report to Dashboard when cancelling user jobs

### ISSUE THREAD

**Issue by ericvaandering** [dmwm/CRAB2#292]
Original Savannah ticket 47309 reported by spiga on Tue Feb 24 04:28:54 2009.

AS was noticed by several users of Dashboard Task Monitoring , cancelled jobs not shown as cancelled in Task monitoring.
I've checked ML server and found out that there was no  report sent from CRAB UI at the point when job had beem canclled.
As a result jobs stay pending or running unless being timeout by Dashboard.
Could  please, Dashboard reporting be enabled in case of job cancellation both for client side submission and server submission. I think this functionalioty was there some time ago, but might have dissapered with time.
Many thanks
Julia

**Comment by ericvaandering** [comment]
Comment by spiga on Thu Nov 19 03:27:10 2009

Hi Julia, 

  which kind of parameter should be used to notify the kill? 

cheers Daniele

**Comment by ericvaandering** [comment]
Comment by julia on Thu Nov 19 03:40:10 2009

Hi Daniele,

This should be standard job status report with status &quot;Cancelled&quot; or &quot;Killed&quot;, same as sent from the UI when CRAB checks status of the job from LB.
I see tahat some crab analysis jobs are marked as cancelled in the Dashboard. Let me find out what message had triggered this status.
Cheers
Julia

**Comment by ericvaandering** [comment]
Comment by spiga on Thu Nov 19 04:05:03 2009

Ok all it's clear to me... there is nothing (I guess) I can do on crab side. Crab (both server and client) reports the right information once there... If you confirm you are able to receive the job status informations, once job can be cancelled you must receive it. THe point is that the killing is not always allowed...so sometime user ask for it but it is not performed..
So basically I'd close this ticket unless you have different opinion.

**Comment by ericvaandering** [comment]
Comment by julia on Thu Nov 19 04:27:57 2009

Fine with me.
I've checked one of cancelled jobs.
It is reported &quot;Cancelled by user&quot; in a standard job status message 
agreement, but later many times reported &quot;Running&quot; again, which I guess is the case, when user for some reason could not kill the job though intended to. So latest reports with 'running status' had been triggered by status returned by LB.
Anyhow, we can not do much about it. What/how it is reported now is fine. The good new is that looking in the last  day statistics I see NO non-resolved jobs from the application point of view among those which were submitted by CRAB server and very few server jobs (2,5%) which have non-resolved GRID status in Dashboard. We are in a pretty good shape. Thanks a lot for your help.
Julia

**Comment by ericvaandering** [comment]
Closed by spiga on Thu Nov 19 07:36:04 2009
